dpdk/src/dev.rs

```diff
@@ -226,7 +226,7 @@ pub struct DevConfig {
     /// Rework this bad idea.
```

Comment:
The commit message has a typo `offloas` instead of `offloads`.

Reply:
sorted